mpmath/matrices/matrices.py

```diff
@@ -993,3 +993,26 @@ def mnorm(ctx, A, p=1):
             return max(ctx.fsum((A[i,j] for j in range(n)), absolute=1) for i in range(m))
         else:
             raise NotImplementedError("matrix p-norm for arbitrary p")
+        
+    def kron(ctx, *args):
+        """
+        Calculates the tensor products of 2 or more matrices, similar to numpy.kron
+        """
+        new_rows: int = 1
+        new_cols: int = 1
```

Comment:
```suggestion
        new_rows = 1
        new_cols = 1
```
We don't include type hints.